libcloudforensics/providers/aws/forensics.py

```diff
@@ -226,6 +226,7 @@ def StartAnalysisVm(
       raise RuntimeError('error - ListImages returns >1 AMI image: [{0:s}]'
                          .format(', '.join(image_names)))
     ami = ami_list[0]['ImageId']
+  assert ami  # Mypy: assert not None
```

Comment:
can you make the assert here and in other places more explicit.
i.e. "assert ami is not None"